src/modules/vendor/components/programs/cards/templates/MultipleChoiceQuestion.vue

```diff
@@ -118,12 +123,14 @@ export default {
       if (this.requiredOneCorrectAnswer(index) || this.removeSingleCorrectAnswer(index)) {
         return 'Une bonne réponse est nécessaire.';
       }
-
+      if (!this.$v.card.qcmAnswers.$each[index].label.maxLength) {
+        return `${QC_ANSWER_MAX_LENGTH} caractères maximum.`;
+      }
       return '';
     },
     answersError (index) {
       return this.requiredAnswerIsMissing(index) || this.requiredOneCorrectAnswer(index) ||
```

Comment:
Ici tu peux eventuellement distinguer les trois conditions anciennes requiredAnswerIsMissing, requiredOneCorrectAnswer, removeSingleCorrectAnswer (comme ona fait dans qcu) qui concernent le nombre de reponse. Et la nouvelle this.$v.card.qcmAnswers.$each[index].$error, qui concerne le nombre de caractere